Assets/HoloToolkit/Input/Scripts/GameControllerManipulator.cs

```diff
@@ -1,15 +1,16 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System;
 using UnityEngine;
 
 namespace HoloToolkit.Unity.InputModule
 {
     /// <summary>
     /// Allows the user to place and rotate GameObjects using a game controller.
-    /// TODO This should be converted to an input source.
     /// </summary>
     /// <remarks>Make sure to enable the HumanInterfaceDevice capability before using.</remarks>
+    [Obsolete]
     public class GameControllerManipulator : MonoBehaviour
```

Comment:
Should we have a message letting people know what they should be using instead of this Obsolete class, or is this just used internally?

Reply:
It was never really working anyway, but I think it's good practice to add the message.